ui/src/main/webapp/app.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import { Provider, connect } from 'react-redux'
+import { ApolloProvider } from 'react-apollo'
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import { Router, hashHistory } from 'react-router'
 
 import store from './store'
+import client from './client'
 
 import Exception from './containers/exceptions'
 import Ldap from './wizards/ldap'
 import Sources from './wizards/sources'
 import { Home } from './home'
-import Wcpm from './adminTools/webContextPolicyManager'
+import Wcpm from './admin-tools/wcpm'
```

Comment:
Can we change the directory structure? `admin-tools` doesn't make much sense to me. Possibly change it to `security` or remove it entirely?